plugin/src/automation/equivalence.mli

```diff
@@ -11,3 +12,22 @@ open Lifting
  * (Don't return the types, since Coq can infer them without issue)
  *)
 val prove_equivalence : env -> evar_map -> lifting -> (types * types)
+
+type pre_adjoint = {
+  orn : lifting;
+  sect : Constant.t;
+  retr0 : Constant.t
+}
+
+(*
+ * Augment the initial retraction proof in order to prove adjunction.
```

Comment:
Please explain this a bit more (why we need this) in the comment in the interface

Reply:
Done. Is the expanded comment detailed enough?